Modules/_multiprocessing/posixshmem.c

```diff
@@ -32,6 +32,7 @@ module _posixshmem
 /*[clinic input]
 _posixshmem.shm_open -> int
     path: unicode
+    /
```

Comment:
BTW, why not use the `/ [3.16]` syntax? These are user-visible changes, right?

Reply:
It's a private module: _posixshmem.

Reply:
Then why the concern about making all the params positional-only?